src/sc/src/controllers/partitions/reducer.rs

```diff
@@ -79,6 +79,7 @@ impl PartitionReducer {
         }
     }
 
+    #[instrument(skip(self, updates))]
```

Comment:
let's put as separate PR.  One for adding feature flag.  And create separate PR for each effected modules.